state-synchronizer/src/tests/integration_tests.rs

```diff
@@ -158,6 +158,8 @@ impl SynchronizerEnv {
         config.base.role = role;
         config.state_sync.sync_request_timeout_ms = timeout_ms;
         config.state_sync.multicast_timeout_ms = multicast_timeout_ms;
+        // Too many tests expect this, so we overwrite the value
+        config.state_sync.chunk_limit = 250;
```

Comment:
does this mean we changed the value to 1000 but we do not exercise it in any tests?

Reply:
smoke tests amongst others, this is only "unit" tests